python/py-ipympl/Portfile

```diff
@@ -21,16 +23,44 @@ long_description    Leveraging the Jupyter interactive widgets framework, \
 
 homepage            https://github.com/matplotlib/jupyter-matplotlib
 
-checksums           rmd160  edabf6803acf97df26b7fe0b24483da64bee70c5 \
-                    sha256  f0f1f356d8cb9d4fb51bb86dbbf837c190145316cb72f66081872ebc4d6db0a1 \
-                    size    56729
+checksums           rmd160  7524cb752caa0423a00cc15b6ec6881a0961c774 \
+                    sha256  49bab75c05673a6881d1aaec5d8ac81d4624f73d292d154c5fb7096f10236a2b \
+                    size    267567
 
 if {${name} ne ${subport}} {
     depends_build-append \
                         port:py${python.version}-setuptools \
+                        port:py${python.version}-wheel \
                         port:py${python.version}-jupyter_packaging
```

Comment:
when making the change above; you should only have the following two dependencies present in `depends_build`:
```
jupyter_packaging~=0.7.0", 
"jupyterlab>=3.0.0,==3.*",
```